### DISCUSSION TITLE
copilot working in latest version?

### DISCUSSION THREAD

**Discussion by Baltrazx** [coder/code-server#5325] [Q&A] [answered]
hey just wanted to ask if github copilot is supported in the latest 4.5 release havent found anything saying it is and i cant test it atm

**Comment by flxcp** [comment, marked as answer]
Yes, GitHub Copilot is compatible and works with **code-server v4.5.0**.

![image](https://user-images.githubusercontent.com/96683372/177598390-406ac0ff-f2b5-4fdb-94e3-4ca518613c77.png)

**Reply by skunkwerk** [reply]
Which version of Copilot are you using, and where are you downloading the VSIX file from?

**Reply by flxcp** [reply]
I am using **Github Copilot v1.34.x**. You can Download the VSIX file from [Marketplace](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot).

**Reply by Baltrazx** [reply]
@flxcp i just installed latest 4.5.1 and when i try to "Install from VSIX" it erros and says copilot not compatible there a way i can check code server version?

**Reply by flxcp** [reply]
You can check the version by running `code-server -v`.

You can go through this https://github.com/coder/code-server/discussions/5393, if you get a symbolic link error.

**Comment by benz0li** [comment]
My advice: Keep your hands off GitHub Copilot.

https://drewdevault.com/2022/06/23/Copilot-GPL-washing.html
